Lib/idlelib/editor.py

```diff
@@ -1013,7 +1013,10 @@ def saved_change_hook(self):
             # that doesn't match platform conventions.
             title = short + " - " + long + _py_version
         elif short:
-            title = short
+            if short == "IDLE Shell":
+                title = short + _py_version
```

Comment:
Parens are noise after this title.  Easy change since platform is already imported.
```suggestion
                title = short + platform.python_version()
```